unlock-app/src/components/content/certification/utils.tsx

```diff
@@ -0,0 +1,27 @@
+import { Metadata } from '~/components/interface/locks/metadata/utils'
+
+interface CertificationUrlProps {
+  metadata?: Partial<Metadata>
+  lockAddress: string
+  network: string | number
+  tokenId?: string | number
+}
+
+export const getCertificationUrl = ({
```

Comment:
Actually since you are returning paths let's use `getCertificationPath` (same for events)